Makefile

```diff
@@ -70,7 +73,66 @@ run-kernel-gateway:
 			$(KG_IMAGE_NAME); \
 	fi;
 
-kill:
+###### tmpnb
+
+MAX_LOG_SIZE:=50m
+MAX_LOG_ROLLOVER:=10
+
+token-check:
+	@test -n "$(TOKEN)" || \
+		(echo "ERROR: TOKEN not defined (make help)"; exit 1)
```

Comment:
Is "TOKEN" name too generic for this usage? Maybe rename to something like `TMPNB_TOKEN`? Or is this token going to be used elsewhere?